partner_classification/readme/DESCRIPTION.md

```diff
@@ -0,0 +1 @@
+Odoo already allows to classify partners through Tags (`res.partner.category`). But this module is a bit more rigid: it only allows for one Classification per Partner. And the Classification is shown at the top of the Form View, under the partner's name.
```

Comment:
this should maybe go into `CONTEXT.md`but that does not describe what this module does really. @thienvh332 Could you improve this a little bit ?

Reply:
Thanks @rousseldenis,
I updated it.